nalgebra-sparse/src/csr.rs

```diff
@@ -190,6 +208,15 @@ impl<T> CsrMatrix<T> {
         }
     }
 
+    /// Return sorted vector.
+    #[inline]
+    #[must_use]
+    pub fn sorted(row_offsets: Vec<usize>) -> Vec<usize> {
+        let mut sorted = row_offsets.clone();
+        sorted.sort();
+        return sorted;
+    }
```

Comment:
this should probably not be a method of `CsrMatrix` as it does not require any access to `self` and seems unrelated to a `CsrMatrix` in general - and certainly not a public method.